app/models/giveaway.rb

```diff
@@ -1,16 +1,23 @@
 class Giveaway < ApplicationRecord
     has_many :users
+    has_many :winners, :class_name => "User" 
     has_many :products
   
     def choose_winners   
-      winners = {} 
       all_applicants = users.to_a.shuffle
+      p all_applicants
```

Comment:
im seeing alot of these prints everywhere. They are fine for testing but I think we should remove them for master